build/input/ImplementationGuide-fhir.us.mcode.json

```diff
@@ -616,9 +616,9 @@
         },
         {
           "reference": {
-            "reference": "ValueSet/ConditionStatusTrendVS"
+            "reference": "ValueSet/CancerStatusTrendVS"
           },
-          "name": "ConditionStatusTrendVS",
+          "name": "CancerStatusTrendVS",
           "description": "Standardized value set that describes the trend or status of a disease, condition, or ability. Based on LOINC answer lists LL2751-7 and LL4721-8. The value SNOMED CT 260415000 (Not Detected) should be used to express No Evidence of Disease (NED), condition resolved, or full remission."
```

Comment:
This description referring to SNOMED CT 260415000 (Not Detected) is out of date with the proposed change.

Reply:
Updated in PR.